packages/drivers/odsp-driver/src/vroom.ts

```diff
@@ -89,8 +89,9 @@ export const fetchJoinSession = mockify(
 				postBody += `X-HTTP-Method-Override: POST\r\n`;
 				postBody += `Content-Type: application/json\r\n`;
 				if (!disableJoinSessionRefresh) {
-					postBody += `prefer: FluidRemoveCheckAccess\r\n`;
+					postBody += `Prefer: FluidRemoveCheckAccess\r\n`;
```

Comment:
why capital P?

Reply:
I think both are accepted, but capitalizing seems more standard, and also used for the other headers above. See also the MDN documentation https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Prefer.